pkg/plugins/platform/local_test.go

```diff
@@ -67,6 +68,15 @@ func TestMain(m *testing.M) {
 	os.Exit(testRun)
 }
 
+// TestLocalSetup reduces the TestSite list on shorter test runs.
+func TestLocalSetup(t *testing.T) {
+	// Allow tests to run in "short" mode, which will only test a single site. This keeps test runtimes low.
+	// We would much prefer to do this in TestMain, but the Short() flag is not yet available at that point.
+	if testing.Short() {
+		TestSites = []testcommon.TestSite{TestSites[0]}
```

Comment:
I will also add that I'd much much prefer to do this in TestMain (and my original code tried to do that), however the testing.Short() always returns false there, regardless of what arguments  you pass to `go test`.